### PR TITLE
Add local repo package resolution integration tests

### LINKED ISSUE #33005: Add test cases for local repository packages
**Description:**
Tests should be added to the ballerina distribution repo.

Scenarios identified: [Doc](https://docs.google.com/document/d/193dui-Wn-cEAL14iRgBwJ0WBujfrT8FNMy19jy8ZDQE/edit?usp=sharing)

**Describe your task(s)**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by Ibaqu**
Fixes #33005

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Comment by pramodya1994** [comment]
Shall we change the PR title to mention the local repo?

File: project-api/project-api-test/src/test/resources/projects_for_resolution_integration_tests/package_aa_1_0_0/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[package]
+org = "adv_res"
+name = "package_aa"
+version = "1.0.0"
+export = ["package_aa"]
```
Comment (pramodya1994) [inline project-api/project-api-test/src/test/resources/projects_for_resolution_integration_tests/package_aa_1_0_0/Ballerina.toml:5]: Can we leave last line as a empty line in these files?

Comment (Ibaqu) [reply project-api/project-api-test/src/test/resources/projects_for_resolution_integration_tests/package_aa_1_0_0/Ballerina.toml]: Added lasntline whereever applicable

**Comment by Ibaqu** [comment]
Closing this pr since one is sent to master

**Pull request closed without merging**